lib/useKResponsiveWindow/MediaQuery.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Class representing a MediaQuery
+ */
+export default class MediaQuery {
+  /**
+   *Create a media query
+   * @param {String} query - The query string
+   * @param {CallableFunction} eventHandler - The event callback function
+   */
+  constructor(query, eventHandler) {
+    this.query = query;
+    this.eventHandler = eventHandler;
+  }
+
+  /**
+   * @returns {Object} Media query list
+   */
+  mediaQueryList() {
```

Comment:
Not needed, but could turn this into a [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) as its only purpose is to return a value. (Would then need to update the references to it to no longer be invocations of a method).